libavcodec/vvc/vvc_ctu.h

```diff
@@ -162,6 +162,7 @@ typedef enum PredFlag {
 } PredFlag;
 
 typedef enum IntraPredMode {
+    INTRA_DEFAULT   = -1,
```

Comment:
invalid is a better name. Because we have no such Intra predict model. 
since it invalid, we fallback to default

Reply:
Done in 6e7b9747700a06ba1427f74aa0050ec70e243338